modules/virt-cluster-resource-requirements.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-cluster-resource-requirements_{context}"]
+= Additional hardware requirements for {VirtProductName}
+
+
+{VirtProductName} is an add-on to {product-title} and imposes additional overhead that you need to take into account when planning a cluster. Each cluster machine must meet the following overheads in addition to the {product-title} requirements.
+
+[id="memory-overhead_{context}"]
+=== Memory overhead
+
+.Cluster memory overhead
+
+----
+Memory overhead per infrastructure node = 158 MiB
+----
+
+----
+Memory overhead per worker node = 366 MiB
+----
+
+Additionally, {VirtProductName} environment resources require a total of 2179 MiB of RAM that is spread across all infrastructure nodes.
+
+
+.Virtual machine memory overhead
+
+----
+Memory overhead per virtual machine = (1.002 * requested memory) + 146 MiB  \
+                + (8 * number of vCPUs) + (16 MiB * number of graphics devices)
+----
+
+Add 1 GiB to the above result if your environment includes a Single Root I/O Virtualization (SR-IOV) network device or a Graphics Processing Unit (GPU).
+
+
+[id="CPU-overhead_{context}"]
+=== CPU overhead
+
+.Cluster CPU overhead
+
+----
+Cluster CPU overhead = (2 * number of worker nodes) + (4 * number of infrastructure nodes) \
+              + CPU overhead per virtual machine
+----
+
+.Virtual machine CPU overhead
+
+CPU overhead per virtual machine is subjective. If dedicated CPUs are requested, there is a 1:1 impact on cluster CPU requirement. Otherwise there is no rule of thumb about how many CPUs a virtual machine would require.
```

Comment:
s/on cluster CPU requirement/on the cluster CPU overhead requirement

s/Otherwise/Otherwise,

s/there is no rule of thumb/there are no specific rules

s/would require/requires

Reply:
Fixed